src/atoms/cc-input-number.js

```diff
@@ -201,11 +230,31 @@ export class CcInputNumber extends LitElement {
         }
 
         label {
+          align-items: flex-end;
           cursor: pointer;
-          display: block;
+          display: flex;
+          gap: 2em;
```

Comment:
this is to make sure the "required" mention and the label text are always at least separated by 2em. Could be set as a margin on `.required` instead but I found it simpler this way.

Reply:
We used to avoid gap on flexbox because of lack of support for Safari but we're one year after the 14.1 and more that 2 versions where released recenlty. That's fine like this :+1:

Reply:
Note that you're using a gap between a text node (the label) and the span required. We should add a span on the `${this.label}` so it's really a flex item (see other comment).